README.md

```diff
@@ -28,31 +28,39 @@ To build the TensorRT OSS components, ensure you meet the following package requ
   * [Python3](https://www.python.org/downloads/release/python-365/) >= v3.6.5
 
 * [PIP](https://pypi.org/project/pip/#history) >= v19.0
+  * PyPI packages
+  * [numpy](https://pypi.org/project/numpy/)
+  * [onnx](https://pypi.org/project/onnx/1.6.0/) 1.6.0
+  * [onnxruntime](https://pypi.org/project/onnxruntime/) >= 1.3.0
+  * [pytest](https://pypi.org/project/pytest/)
 
 * Essential libraries and utilities
   * [Git](https://git-scm.com/downloads), [pkg-config](https://www.freedesktop.org/wiki/Software/pkg-config/), [Wget](https://www.gnu.org/software/wget/faq.html#download), [Zlib](https://zlib.net/)
 
 * Cross compilation for Jetson platforms requires JetPack's host component installation
-  * [JetPack](https://developer.nvidia.com/embedded/jetpack) >= 4.2
+  * [JetPack](https://developer.nvidia.com/embedded/jetpack) >= 4.4
 
 **Optional Packages**
 
 * Containerized builds
-  * [Docker](https://docs.docker.com/install/) >= 1.12
-  * [NVIDIA Docker](https://github.com/NVIDIA/nvidia-docker) >= 2.0
+  * [Docker](https://docs.docker.com/install/) >= 19.03
+  * [NVIDIA Docker](https://github.com/NVIDIA/nvidia-docker) >= 2.0 or `nvidia-container-toolkit`
 
 * Code formatting tools
   * [Clang-format](https://clang.llvm.org/docs/ClangFormat.html)
   * [Git-clang-format](https://github.com/llvm-mirror/clang/blob/master/tools/clang-format/git-clang-format)
 
+* Required PyPI packages for Demos
+  * [Tensorflow-gpu](https://pypi.org/project/tensorflow/1.14.0/) == 1.15.0
+
 **TensorRT Release**
 
-* [TensorRT](https://developer.nvidia.com/nvidia-tensorrt-download) v7.0
+* [TensorRT](https://developer.nvidia.com/nvidia-tensorrt-download) v7.1
 
 
 NOTE: Along with the TensorRT OSS components, the following source packages will also be downloaded, and they are not required to be installed on the system.
 
-- [ONNX-TensorRT](https://github.com/onnx/onnx-tensorrt) v7.0
+- [ONNX-TensorRT](https://github.com/onnx/onnx-tensorrt) v7.1
 - [CUB](http://nvlabs.github.io/cub/) v1.8.0
 - [Protobuf](https://github.com/protocolbuffers/protobuf.git) v3.8.x
```

Comment:
Pretty sure this should protobuf version is v3.0.0 per https://github.com/NVIDIA/TensorRT/pull/378 and https://github.com/NVIDIA/TensorRT/pull/654/files#diff-af3b638bc2a3e6c650974192a53c7291R85